pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.580.1</version>
+        <version>1.607</version>
```

Comment:
1.580.1 was an LTS so this is fine. If there is no particular reason to require a newer LTS line we generally do not do so.